### PR TITLE
proposal for user control of skaffold helpers

### PR DESCRIPTION
**Pull request by loosebazooka**
fyi @lightoze @balopat 

rendered: https://github.com/GoogleContainerTools/jib/blob/skaffold-config-proposal/proposals/skaffold_config.md

Should help with: https://github.com/GoogleContainerTools/skaffold/issues/3457

### CODE REVIEW

**Comment by chanseokoh** [comment]
This probably works. But I wonder if other Skaffold builders eventually need this kind of stuff. If that's the case, it'd be better to have a general support in Skaffold?

**Comment by loosebazooka** [comment]
Um, so yeah there's that, but I think this helps with portability of jib across skaffold files.

File: proposals/skaffold_config.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      exclude = 'a/file'
+    }
+  }
+}
+```
+
+#### Maven (`pom.xml`)
+```xml
+<configuration>
+  <outputPaths>
```
Comment (TadCordle) [inline proposals/skaffold_config.md]: Shouldn't be in `<outputPaths>`.

File: proposals/skaffold_config.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <watch>
+        <includes>some/file,another/file</includes>
+        <excludes>
+          <exclude>not/me</exclude>
+          <exclude>also/not/me</exclude>
+        <excludes>
+      </watch>
+      <sync>
+        <excludes>some/file<excludes>
+    </skaffold>
```
Comment (TadCordle) [inline proposals/skaffold_config.md]: Missing `</sync>`

File: proposals/skaffold_config.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Proposal: Control over skaffold tasks/goals
+
+Relevant issues: https://github.com/GoogleContainerTools/skaffold/issues/3457,
```
Comment (TadCordle) [inline proposals/skaffold_config.md]: If there's only one,
```suggestion
Relevant issue: https://github.com/GoogleContainerTools/skaffold/issues/3457
```

File: proposals/skaffold_config.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      exclude = 'a/file'
+    }
+  }
+}
+```
+
+#### Maven (`pom.xml`)
+```xml
+<configuration>
+  <skaffold>/absolute/location.tar
```
Comment (briandealwis) [inline proposals/skaffold_config.md]: What does `/absolute/location.tar` mean?

File: proposals/skaffold_config.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    <watch>
+      <buildIncludes>
+        <buildInclude>some/pomfile.xml</buildInclude>
+      </buildIncludes>
+      <includes>
+        <include>some/file</include>
+        <include,another/file</include>
```
Comment (briandealwis) [inline proposals/skaffold_config.md]: typo: `,` &rarr; `>`

File: proposals/skaffold_config.md (hunk 4 above)
Comment (loosebazooka) [reply proposals/skaffold_config.md]: man, I really messed up this doc, I'll push an update.

**Pull request merged**